package.json

```diff
@@ -15,5 +15,8 @@
   },
   "devDependencies": {
     "concurrently": "^8.2.2"
+  },
+  "dependencies": {
```

Comment:
we do not add any front-end or server-end dependecies into the root package.json. we can have only those dependecnies here that are shared between both yarn workspaces: client and server. please move it to client, if needed.